src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeRevurderFraValidering.kt

```diff
@@ -0,0 +1,60 @@
+package no.nav.tilleggsstonader.sak.vilkår.vilkårperiode
+
+import no.nav.tilleggsstonader.libs.log.SecureLogger.secureLogger
+import no.nav.tilleggsstonader.sak.behandling.domain.Saksbehandling
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvis
+import no.nav.tilleggsstonader.sak.util.norskFormat
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.Vilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.dto.LagreVilkårperiode
+import java.time.LocalDate
+
+object VilkårperiodeRevurderFraValidering {
+
+    fun validerNyPeriodeIForholdTilRevurderFra(
```

Comment:
```suggestion
    fun validerKanLeggeTilNyPeriodeRevurdering(
```

Evt. om det ikke er noe mer enn fom som må sjekkes på denne: 

```suggestion
    fun validerNyPeriodeFomEtterRevurderFra(
```

Reply:
Innså nå at fom i siste kanskje ble litt overflødig